site/index.html

```diff
@@ -11,7 +11,7 @@
   content: Sonobuoy is a diagnostic tool that makes it easier to understand the state of a Kubernetes cluster by running a choice of configuration tests in an accessible and non-destructive manner.
   cta_link1:
     text: Get Started with Sonobuoy
-    url: /docs/v0.14.3
+    url: /docs/v0.16.4
```

Comment:
Thanks for finding this; I didnt realize the old version was hardcoded.

Just remove the version and leave the link as `/docs` and it will keep sending you to the newest released version (instead of v0.16.4 becoming outdated and this needing updated again)